pages/osx/duti.md

```diff
@@ -0,0 +1,27 @@
+# duti
+
+> Set default applications for document types and URL schemes on macOS.
+
+- Set Safari as the default handler for HTML documents:
+
+`duti -s com.apple.Safari public.html all`
```

Comment:
I didn't feel it was appropriate since the syntax in the given example is specific, and I thought tokenized syntax was reserved for more ambiguous parameters. Alternatively, we could provide and example like

```
- Set a program as the default handler for a given extension:

`duti -s {{application_id}} {{public.ext}} all`
```

But I don't think that would help anyone looking for a tl;dr, since it's not immediately clear what an `application_id` is, and `duti` doesn't provide a way to lookup application IDs without referencing file extensions, UTIs, or URL schemes.

Reply:
Tokens does not mean you have to remove concrete info. They just signify user provided values. Let us change it to - {{com.apple.Safari}} {{public.html}}